spec/security_considerations.md

```diff
@@ -3,106 +3,75 @@ There are many security considerations related to web requests, storing informat
 
 ### Common security threats
 
-`did:webs` strives to narrow the attack surface against common threats, such as:
-- Broken Object Level Authorization (BOLA)
-- Denial of service (DoS) attack
-- Deletion attack
-- Duplicity detection
-- Eclipse attack
-- Forgery
-- Impersonation attack
-- Key Compromise attack
-- Malleability attack
-- Replay attack
+1. All `did:webs` features MUST narrow the attack surface against common threats, such as:
```

Comment:
"narrow" => "reduce or protect"?

Reply:
@daidoji thank you for review. I have accepted the suggestions and agree that 'reduce' is better and have changed that now.